lib-messagebus/platforms/platformconfig.toml

```diff
@@ -21,10 +21,8 @@
 # configuration management.
 
 [KAFKA]
-# Kafka Server List.
-KServers   = ""
-# Listening port.
-KLPort     = 9092
+# Kafka Server List .
```

Comment:
A bit more info like below, about the value expected for parameter will be helpful.
KServersInfo defines the list of Kafka Server URI/Nodename:port. Example: KServersInfo = [localhost:9092].

Reply:
updated